Assets/_ProjectAssets/Scripts/Utilities/CheckIfSelectedShopItemIsVisible.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class CheckIfSelectedShopItemIsVisible : MonoBehaviour
+{
+    private void Update()
+    {
+        if (transform.position.y>3.65f)
+        {
+            transform.GetChild(0).gameObject.SetActive(false);
+        }
+        else
+        {
+            transform.GetChild(0).gameObject.SetActive(true);
+        }
+
```

Comment:
Ce plm  e asta si de ce e in Update? si de ce ai nevoie de o clasa pt asta?

Reply:
e un cacat care sa nu lese particulele din shop sa fie vizibile cand se ascund dupa meniul cela de sus
ca ele apar deasupra, si aici fac check ca daca elementul e prea sus sa nu mai emita particule ca arata de cacat

Reply:
setezi layer-u de la particule system mai mic decat cel de la shop si gata

Reply:
daca il pun pe 0 nu se vede deloc, daca il pun pe 1 e mereu vizibil
nu e nimic intre ca sa pot seta

Reply:
Inseamna ca ai alte obiecte pe acelasi layer care se suprapun cu particulele

Reply:
-pai am layerul UI pe care sta tot UI-ul
nu  pot sa scot top-bar-ul cela din UI ca-i UI

Reply:
ce zici tu e layer pt physics ce zic eu e layer-ul pt randare

Reply:
pai da cum schimb layerul de randare la un UI?
ca cand il schimb la particule ori e vizibil mereu ori nu e vizibil deloc